src/main/java/tatracker/logic/commands/session/DeleteSessionCommand.java

```diff
@@ -62,6 +62,11 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         Session sessionToDelete = lastShownList.get(index.getZeroBased());
+        if (sessionToDelete.getIsRecurring()) {
+            RecurSessionCommand recurSession = new RecurSessionCommand(sessionToDelete);
+            recurSession.createRecurSession();
+        }
```

Comment:
This would be better suited for the done session command rather than the delete session command as when a user is deleting a session, they'd want that session to be deleted. However, when the session is done, they'd want it to be back in their session list for say, one week from now. We will probably have to explain this in the UG too.